pywbem/cim_obj.py

```diff
@@ -2511,6 +2511,72 @@ def to_wbem_uri(self, format='standard'):
 
         return _ensure_unicode(''.join(ret))
 
+    @staticmethod
+    def from_instance(class_, instance, namespace=None, host=None,
+                      strict=False):
+        """
+        Given a class and corresponding instance, create and return the
+        CIMInstanceName from the class key properties and instance key
+        property values.
+
+        If a property value does not exist in the instance it gets the
+        value from the class default_value unless strict is set. If strict
+        is True, all key properties must exist in the instance
+
+        Parameters:
+
+            class_ (:class:~pywbem.CIMClass`):
+                CIMClass from which the key properties will be retrieved.
+
+            instance: (:class:~pywbem.CIMInstance`):
```

Comment:
The colon directly after the parameter name is incorrect. Occurs multiple times.

Reply:
Done